CSharp/Library/Microsoft.Bot.Connector.Shared/JwtConfig.cs

```diff
@@ -11,7 +11,7 @@ public static class JwtConfig
         /// <summary>
         /// TO CHANNEL FROM BOT: Login URL
         /// </summary>
-        public const string ToChannelFromBotLoginUrl = "https://login.microsoftonline.com/botframework.com/oauth2/v2.0/token";
+        public const string ToChannelFromBotLoginUrlFormat = "https://login.microsoftonline.com/{0}/oauth2/v2.0/token";

```

Comment:
Can you add a comment: botframework.com is the default tenant that applies in most cases